pkg/deploy/config.go

```diff
@@ -189,6 +194,25 @@ func UnmarshalFeatureMap(data string) (FeatureMap, error) {
 	return i, nil
 }
 
+// UnmarshalFeatureMap parses the JSON configuration data and returns
+// an initialised FeatureMap. If `data` is an empty string an empty
+// FeatureMap is returned.
+func UnmarshalNotificationsMap(data string) (*NotificationsMap, error) {
```

Comment:
rename along with the Notifications type
```suggestion
func UnmarshalNotifications(data string) (*NotificationsMap, error) {
```